proposals/csharp-8.0/using.md

```diff
@@ -118,16 +118,8 @@ using (var r = new Resource())
 }
 ```
 
-This will allow developers to leverage `using` in a number of new scenarios:
-
-- `ref struct`: These types can't implement interfaces today and hence can't participate in `using`
+This will allow developers to leverage `using` for `ref struct` types. These types can't implement interfaces today and hence can't participate in `using`
 statements.
-- Extension methods will allow developers to augment types in other assemblies to participate 
```

Comment:
I confirmed from the code (`TryFindDisposePatternMethod`) that extension methods don't count. Tagging @chsienki to confirm this was intentional and to fix the speclet.

Reply:
Yes, extension methods we're explicitly disallowed, even for ref structs. The decision being that it would have been legal to have an extension method called 'Dispose' that previously wouldn't have been called if you used the type in a foreach. With the extension method lookup, it would now get called, which was considered too big a breaking change. 

LDM notes are here: https://github.com/dotnet/csharplang/blob/3c8559f186d4c5df5d1299b0eaa4e139ae130ab6/meetings/2018/LDM-2018-12-12.md#when-we-start-recognizing-pattern-dispose-do-we-call-it-in-foreach